.eslintrc

```diff
@@ -71,6 +74,22 @@
         "no-underscore-dangle": 0,
         "no-use-before-define": 2,
         "quotes": [2, "single"],
+        "react/display-name": 1,
+        "react/jsx-boolean-value": 1,
+        "react/jsx-quotes": 1,
+        "react/jsx-no-undef": 1,
+        "react/jsx-sort-props": 1,
+        "react/jsx-sort-prop-types": 1,
+        "react/jsx-uses-react": 1,
+        "react/jsx-uses-vars": 1,
+        "react/no-did-mount-set-state": 1,
+        "react/no-did-update-set-state": 1,
+        "react/no-multi-comp": 1,
+        "react/no-unknown-property": 1,
+        "react/prop-types": 1,
+        "react/react-in-jsx-scope": 1,
+        "react/self-closing-comp": 1,
+        "react/wrap-multilines": 1,
```

Comment:
Defaulting these to warnings to avoid breaking changes. Next major bump may make them errors.